### DISCUSSION TITLE
File the InternationalizeDemo.zip is missing

### DISCUSSION THREAD

**Discussion by maxrexfax** [apache/netbeans#6471] [General]
### Description

On page https://netbeans.apache.org/kb/docs/java/gui-automatic-i18n.html there is an url to file InternationalizeDemo.zip
On the page https://netbeans.org/files/documents/4/770/InternationalizeDemo.zip we can read PAGE NOT FOUND.

### Are you willing to submit a pull request?

No

**Comment by chawglendale** [comment]
I also encounter the same problem with [GapSupport.zip](https://netbeans.org/projects/samples/downloads/download/Samples%252FJava%252FGapSupport.zip)

**Comment by ebarboni** [comment]
Should be fixed by the following. But this is outdated material I guess.
https://github.com/apache/netbeans-antora-tutorials/pull/10
https://github.com/apache/netbeans-antora-tutorials/pull/9
